dev/io.openliberty.microprofile.reactive.messaging.internal_fat/fat/src/io/openliberty/microprofile/reactive/messaging/fat/telemetry/ReactiveMessagingTelemetryTest.java

```diff
@@ -0,0 +1,105 @@
+import static com.ibm.ws.microprofile.reactive.messaging.fat.kafka.common.KafkaUtils.kafkaClientLibs;
+import static com.ibm.ws.microprofile.reactive.messaging.fat.kafka.common.KafkaUtils.kafkaPermissions;
+import static com.ibm.ws.microprofile.reactive.messaging.fat.kafka.framework.KafkaTestClientProvider.CONNECTION_PROPERTIES_KEY;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.PropertiesAsset;
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.ws.microprofile.reactive.messaging.fat.kafka.common.ConnectorProperties;
+import com.ibm.ws.microprofile.reactive.messaging.fat.kafka.common.KafkaTestConstants;
+import com.ibm.ws.microprofile.reactive.messaging.fat.kafka.common.KafkaUtils;
+import com.ibm.ws.microprofile.reactive.messaging.fat.kafka.framework.AbstractKafkaTestServlet;
+import com.ibm.ws.microprofile.reactive.messaging.fat.kafka.framework.KafkaTestClientProvider;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.TestServlet;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.rules.repeater.RepeatTests;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+import io.openliberty.microprofile.reactive.messaging.fat.apps.telemetry.ReactiveMessagingTelemetryTestServlet;
+import io.openliberty.microprofile.reactive.messaging.fat.apps.telemetry.RmTelemetryProcessingBean;
+import io.openliberty.microprofile.reactive.messaging.fat.apps.telemetry.RmTelemetryReceptionBean;
+import io.openliberty.microprofile.reactive.messaging.fat.suite.KafkaTests;
+import io.openliberty.microprofile.reactive.messaging.fat.suite.ReactiveMessagingActions;
+
+/**
+ * Assert what happens when a message from Kafka is nacked
+ */
+@RunWith(FATRunner.class)
+public class ReactiveMessagingTelemetryTest extends FATServletClient {
+
+    private static final String APP_NAME = "ReactiveMessagingTelemetryApp";
+    public static final String SERVER_NAME = "RxMessagingServerWithTelemetry";
+    private static final String TOPIC_NAME = "RMTelemetryTopic";
+
+    @Server(SERVER_NAME)
+    @TestServlet(contextRoot = APP_NAME, servlet = ReactiveMessagingTelemetryTestServlet.class)
+    public static LibertyServer server;
+
+    @ClassRule
+    public static RepeatTests r = ReactiveMessagingActions.repeat(SERVER_NAME,
+                                                                  ReactiveMessagingActions.MP61_RM30,
+                                                                  ReactiveMessagingActions.MP50_RM30,
+                                                                  ReactiveMessagingActions.MP60_RM30);
```

Comment:
I would recommend removing the MP60_RM30 repeat, there is no particular element of Config that these tests would be exercised that isn't covered elsewhere, this will help reduce the total runtime of the bucket in Full Mode